### PR TITLE
Pet page - basic page was added, Router structure was fixed in order make header and footer available from each page

### PR DESCRIPTION
**Pull request by elinaKM**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      <SideDrawerMenu
+        isOpen={isMenuVisible}
+        onCloseMenu={sideDrawerCloseHandler}
+      />
+      {isMenuVisible && <Backdrop clickHandler={sideDrawerCloseHandler} />}
+      <Header setMenuVisible={setMenuVisible} />
+      <Content>
+        <Router>
+          <Home path="petfinder" />
+          <PetPage path="petfinder/pets/:petId" />
```
Comment (elinaKM) [inline src/App.js]: עבירי סטרינגים של ניווט לקבועים

File: src/components/Wellcome.js [hunk 2]
```diff
@@ -1,26 +1,22 @@
...
                 <Title>
                     Find your Best Friend
                 </Title>
                 <Text>
                     This site is a perfect place to find a perfect pet.<br />
                     Animals bring a lot of light and joy to our life.<br />
-                    Pick one.<br />
-                    Save one live.<br />
-                    Find a new love.<br />
+                    Pick one. Save one live. Find a new love.
```
Comment (okvv) [inline src/components/Wellcome.js]: why not another <text> instead of BR?

File: src/components/common/InputWithLabel.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
```
Comment (okvv) [inline src/components/common/InputWithLabel.js]: LabledInput?

File: src/components/common/TextAreaWithLabel.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
```
Comment (okvv) [inline src/components/common/TextAreaWithLabel.js]: LabledTextArea?

File: src/pages/PetPage.js [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+                    info: "Vaccinations up to date, spayed / neutered."
+                }
+            ],
+            org: "CARMA, Fredericton Chapter"
+            }
+}
+
+const PetPage = (props) => {
+    
+    const {imgs, name, breed, location, age, gender, size, color, aboutInfo, org} = petsDB[props.petId];
```
Comment (okvv) [inline src/pages/PetPage.js]: add defaults or it'll crash

**Review by okvv — CHANGES_REQUESTED** [review]

File: src/components/common/InputWithLabel.js (hunk 3 above)
Comment (elinaKM) [reply src/components/common/InputWithLabel.js]: done

File: src/App.js (hunk 1 above)
Comment (elinaKM) [reply src/App.js]: done

File: src/components/common/TextAreaWithLabel.js (hunk 4 above)
Comment (elinaKM) [reply src/components/common/TextAreaWithLabel.js]: done

File: src/components/Wellcome.js (hunk 2 above)
Comment (elinaKM) [reply src/components/Wellcome.js]: done

File: src/pages/PetPage.js (hunk 5 above)
Comment (elinaKM) [reply src/pages/PetPage.js]: done

**Pull request merged**